test/index.js

```diff
@@ -107,6 +107,7 @@ describe('creditCardType', function () {
       ['18002', 'jcb'],
       ['3530111333300000', 'jcb'],
       ['3566002020360505', 'jcb'],
+      ['3537286818376838569', 'jcb'],
```

Comment:
Can we add tests for jcb cards of length 17 and 18 as well?

Reply:
Is it confirmed that we'll have valid lengths of 17 and 18?

Reply:
Yes.

Reply:
Ah, only saw your previous comment now. Will update!